### PR TITLE
[GameController] Add support for xcode 13 beta3.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/gamecontroller.cs [hunk 1]
```diff
@@ -2018,7 +2042,7 @@ interface GCVirtualController
 		[NullAllowed, Export ("controller", ArgumentSemantic.Weak)]
 		GCController Controller { get; }
 
-		[Export ("changeElement:configuration:")]
+		[Export ("updateConfigurationForElement:configuration:")]
 		void Change (string element, GCVirtualControllerElementUpdateBlock configuration);
```
Comment (rolfbjarne) [inline src/gamecontroller.cs:2046]: Maybe rename the managed method too?

```suggestion
		void UpdateConfiguration (string element, GCVirtualControllerElementUpdateBlock configuration);
```

**Review by rolfbjarne — APPROVED** [review]

File: src/gamecontroller.cs (hunk 1 above)
Comment (spouliot) [reply src/gamecontroller.cs:2046]: in doubt check the swift name :)

**Review by spouliot — APPROVED** [review]

**Comment by mandel-macaque** [comment]
Test failure is unrelated.

**Review by rachelkang — APPROVED** [review]

**Pull request merged**